### DISCUSSION TITLE
Bug

### DISCUSSION THREAD

**Discussion by Ali3447** [farcasterxyz/protocol#143] [General]
{
  "error": "\n######################################################################\n\nUnhandledFetchError\n...message: Cannot read property 'map' of undefined\n...name: Farcaster API Error\n...absoluteUrl: https://client.warpcast.com/v2/user-preferences\n...body: undefined\n...endpointName: getUserPreferences\n...hasTimedOut: false\n...isNetworkError: false\n...method: GET\n...relativeUrl: /v2/user-preferences\n...resolvedTimeout: 20000\n...response: [object Object]\n...responseData: undefined\n...status: 502\n...timeout: undefined\n...componentStack: \n    in ErrorBoundary\n    in ApiTokenAuthErrorBoundary\n    in Unknown\n    in Unknown\n    in Unknown\n    in Unknown\n    in Unknown\n    in Unknown\n    in InvalidateAndPurgeQueryCachesOnStartup\n    in Unknown\n    in ErrorBoundary\n    in QueryErrorResetBoundary\n    in RetryableErrorBoundary\n    in QueryClientProvider\n    in Suspense\n    in RNCSafeAreaProvider\n    in SafeAreaProvider\n    in Unknown\n    in MenuUnderlayProvider\n    in Unknown\n    in Unknown\n    in Unknown\n    in Unknown\n    in Unknown\n    in Unknown\n    in Unknown\n    in Unknown\n    in RootToastProvider\n    in ToastProvider\n    in Unknown\n    in Unknown\n    in Unknown\n    in AndroidNavigationBarColor\n    in Unknown\n    in Unknown\n    in Unknown\n    in Unknown\n    in AnalyticsEnabledProvider\n    in Unknown\n    in Unknown\n    in Unknown\n    in Unknown\n    in ErrorBoundary\n    in DebugLogsProvider\n    in ErrorHistoryProvider\n    in Unknown\n    in RCTView\n    in Unknown\n    in GestureHandlerRootView\n    in gestureHandlerRootHOC(App)\n    in RCTView\n    in Unknown\n    in RCTView\n    in Unknown\n    in AppContainer\n...isComponentError: true\n...unwrappedErrorMessage: Cannot read property 'map' of undefined\n...unwrappedErrorStack: TypeError: Cannot read property 'map' of undefined\n    at ?anon_0_ (address at /var/mobile/Containers/Data/Application/70BFDD82-CE89-408B-A134-CEE1677579A2/Library/Application Support/.expo-internal/52316bd017047cf845fed789603cdd4b.bundle:1:1515243)\n    at next (native)\n    at asyncGeneratorStep (address at /var/mobile/Containers/Data/Application/70BFDD82-CE89-408B-A134-CEE1677579A2/Library/Application Support/.expo-internal/52316bd017047cf845fed789603cdd4b.bundle:1:646554)\n    at _next (address at /var/mobile/Containers/Data/Application/70BFDD82-CE89-408B-A134-CEE1677579A2/Library/Application Support/.expo-internal/52316bd017047cf845fed789603cdd4b.bundle:1:646825)\n    at tryCallOne (address at /var/mobile/Containers/Data/Application/70BFDD82-CE89-408B-A134-CEE1677579A2/Library/Application Support/.expo-internal/52316bd017047cf845fed789603cdd4b.bundle:1:344161)\n    at anonymous (address at /var/mobile/Containers/Data/Application/70BFDD82-CE89-408B-A134-CEE1677579A2/Library/Application Support/.expo-internal/52316bd017047cf845fed789603cdd4b.bundle:1:344837)\n    at apply (native)\n    at anonymous (address at /var/mobile/Containers/Data/Application/70BFDD82-CE89-408B-A134-CEE1677579A2/Library/Application Support/.expo-internal/52316bd017047cf845fed789603cdd4b.bundle:1:354309)\n    at _callTimer (address at /var/mobile/Containers/Data/Application/70BFDD82-CE89-408B-A134-CEE1677579A2/Library/Application Support/.expo-internal/52316bd017047cf845fed789603cdd4b.bundle:1:353306)\n    at _callReactNativeMicrotasksPass (address at /var/mobile/Containers/Data/Application/70BFDD82-CE89-408B-A134-CEE1677579A2/Library/Application Support/.expo-internal/52316bd017047cf845fed789603cdd4b.bundle:1:353470)\n    at callReactNativeMicrotasks (address at /var/mobile/Containers/Data/Application/70BFDD82-CE89-408B-A134-CEE1677579A2/Library/Application Support/.expo-internal/52316bd017047cf845fed789603cdd4b.bundle:1:355384)\n    at __callReactNativeMicrotasks (address at /var/mobile/Containers/Data/Application/70BFDD82-CE89-408B-A134-CEE1677579A2/Library/Application Support/.expo-internal/52316bd017047cf845fed789603cdd4b.bundle:1:208345)\n    at anonymous (address at /var/mobile/Containers/Data/Application/70BFDD82-CE89-408B-A134-CEE1677579A2/Library/Application Support/.expo-internal/52316bd017047cf845fed789603cdd4b.bundle:1:207487)\n    at __guard (address at /var/mobile/Containers/Data/Application/70BFDD82-CE89-408B-A134-CEE1677579A2/Library/Application Support/.expo-internal/52316bd017047cf845fed789603cdd4b.bundle:1:208222)\n    at flushedQueue (address at /var/mobile/Containers/Data/Application/70BFDD82-CE89-408B-A134-CEE1677579A2/Library/Application Support/.expo-internal/52316bd017047cf845fed789603cdd4b.bundle:1:207398)\n    at invokeCallbackAndReturnFlushedQueue (address at /var/mobile/Containers/Data/Application/70BFDD82-CE89-408B-A134-CEE1677579A2/Library/Application Support/.expo-internal/52316bd017047cf845fed789603cdd4b.bundle:1:207341)\n\n######################################################################\n",
  "localTimestamp": 1707276119378,
  "unauthed": true
}

**Comment by horsefacts** [comment]
Hi there—this looks like an error from the Warpcast client. If you're still having trouble, you can email [support@merklemanufactory.com](mailto:support@merklemanufactory.com).

This discussion forum is for the Farcaster protocol, so I'm going to close this conversation. Please reach out to the support email above if you need help with Warpcast.
